yarn.lock

```diff
@@ -55,8 +55,8 @@ ajv-keywords@^1.0.0:
   resolved "https://registry.yarnpkg.com/ajv-keywords/-/ajv-keywords-1.1.1.tgz#02550bc605a3e576041565628af972e06c549d50"
 
 ajv@^4.7.0:
-  version "4.8.2"
-  resolved "https://registry.yarnpkg.com/ajv/-/ajv-4.8.2.tgz#65486936ca36fea39a1504332a78bebd5d447bdc"
+  version "4.9.0"
```

Comment:
I think it's part of a project hygiene to call `yarn upgrade` on each commit. Thanks to semver, this will only upgrade to modules that are compatible (no API or breaking changes).